modules/oxide/src/less/skins/ui/tinymce-5/skin.less

```diff
@@ -111,8 +116,14 @@
     background: @toolbar-background !important;
   }
 
+  .tox-menubar + .tox-toolbar,
```

Comment:
This rule will never match, as you have the same rule below that will always override it.

Reply:
Great eye! Fixed